core/sql/sqlcomp/CmpSeabaseDDLtable.cpp

```diff
@@ -4275,7 +4275,10 @@ short CmpSeabaseDDL::dropSeabaseTable2(
   short *lobNumList = new (STMTHEAP) short[numCols];
   short *lobTypList = new (STMTHEAP) short[numCols];
   char  **lobLocList = new (STMTHEAP) char*[numCols];
-  const char *lobHdfsServer = CmpCommon::getDefaultString(LOB_HDFS_SERVER);
+
+  char  lobHdfsServer[256] ; // max length determined by dfs.namenode.fs-limits.max-component-length(255)
+  memset(lobHdfsServer,0,256);
+  strncpy(lobHdfsServer,CmpCommon::getDefaultString(LOB_HDFS_SERVER),sizeof(lobHdfsServer));
```

Comment:
Just a small, optional, addition: We could also get rid of the (char *) lobHdfsServer cast on line 4334 below, now that lobHdfsServer is a non-const. Same for line 5538 in file NATable.cpp.